src/arch/ArchHooks/ArchHooks_Win32.cpp

```diff
@@ -148,20 +150,12 @@ void ArchHooks_Win32::BoostPriority()
 	 * in the background.  We don't really want to be high-priority--above normal should
 	 * be enough.  However, ABOVE_NORMAL_PRIORITY_CLASS is only supported in Win2000
 	 * and later. */
-	OSVERSIONINFO version;
```

Comment:
I'm not quite sure what's happening here and below. Can you explain a bit why removing this is good + modifying the condition below?

Reply:
`GetVersionEx` is deprecated, and doesn't work properly with newer versions of Windows, based on some documentation that I read. With the exception of one place where I suppressed the deprecation warning (I couldn't find a good alternative), I tried to replace this function with something more modern.

I believe my changes to `GetVideoDriverInfo()` technically dropped support for Windows 95...but I really hope we're not intending to support versions of Windows that are that old anyways.

Reply:
Lol we are not supporting W95 so you're good there. Thanks for the explanation!